src/Public/OpenApi.ps1

```diff
@@ -116,15 +121,31 @@ function Enable-PodeOpenApi {
         [int]
         $Depth = 20,
 
-        [switch]
-        $DisableMinimalDefinitions
+        [switch ]
+        $DisableMinimalDefinitions,
+
+        [switch ]
+        $EnableMinimalDefinitions
+
     )
+    if ( $EnableMinimalDefinitions.IsPresent -and $DisableMinimalDefinitions.IsPresent) {
+        throw 'Properties EnableMinimalDefinitions and DisableMinimalDefinitions are mutually exclusive'
+    }
 
     if ($PSCmdlet.ParameterSetName -eq 'Deprecated') {
         Write-PodeHost -ForegroundColor Yellow "WARNING: The parameter Title,Version and Description are deprecated. Please use 'Add-PodeOAInfo' instead."
+        $PodeContext.Server.OpenAPI.hiddenComponents.enableMinimalDefinitions = $false
+    } else {
+        $PodeContext.Server.OpenAPI.hiddenComponents.enableMinimalDefinitions = $true
     }
 
-    $PodeContext.Server.OpenAPI.hiddenComponents.enableMinimalDefinitions = !$DisableMinimalDefinitions.ToBool()
+    if ($DisableMinimalDefinitions.IsPresent) {
+        $PodeContext.Server.OpenAPI.hiddenComponents.enableMinimalDefinitions = $false
+    }
+
+    if ($EnableMinimalDefinitions.IsPresent) {
+        $PodeContext.Server.OpenAPI.hiddenComponents.enableMinimalDefinitions = $true
+    }
```

Comment:
I don't understand why both Disable and Enable are needed, if the Disable switch isn't supplied then Enable is just assume by default?

The now setting of `enableMinimalDefinitions = $false` when a Deprecated parameter is supplied, requiring `EnableMinimalDefinitions` to now be supplied, is a breaking change for anybody that updates to the next version.

Reply:
This one was a tentative 
But I think I messed up  😊
It was too late at night i'm going to revert to the previous code

Reply:
reverted